src/main/java/cz/smarteon/loxone/app/state/DigitalState.java

```diff
@@ -0,0 +1,44 @@
+package cz.smarteon.loxone.app.state;
+
+/**
+ * Enum containing the different switch states for following controls.
+ * <ul>
+ *     <li><code>SwitchControl</code></li>
+ *     <li><code>DigitalInfoControl</code></li>
+ * </ul>
+ */
+public enum DigitalState {
```

Comment:
Well, I still see little value in this class. In case of analog there is a Double which is nul when not initialized or having a value when initialized. Why in case of digital there can't be `Boolean` which is null when not initialized.

Or vice versa, why there is no corresponding AnalogState class.

Please consider to have this consistent.

Reply:
Okay makes sense i've used a Boolean to align it with the AnalogState